src/sentry/snuba/metrics/helpers.py

```diff
@@ -492,12 +492,9 @@ def _build_queries_for_entity(self, query_definition, entity, fields, where, gro
             orderby=self._build_orderby(query_definition, entity),
         )
 
-        if totals_query.orderby is None:
-            series_query = totals_query.set_groupby(
-                (totals_query.groupby or []) + [Column(TS_COL_GROUP)]
-            )
-        else:
-            series_query = None
+        series_query = totals_query.set_groupby(
+            (totals_query.groupby or []) + [Column(TS_COL_GROUP)]
+        )
```

Comment:
We need to multiple `series_query.limit` by the number of intervals here as well, right? For the case that only a single field is requested and we hit the `else` branch here: https://github.com/getsentry/sentry/pull/32148/files#diff-fe042572eb7930f0cd4d0ce5b501b92fe1fd200e29a24b86004245745d8e8812R420

Reply:
Wouldn't that just default to the `MAX_POINTS` .. the logic you added in your previous PR?

Reply:
Either way, I think your suggestion makes sense so added modification

Reply:
As a follow up PR, I am thinking maybe we should revert this https://github.com/getsentry/sentry/pull/31918